### PR TITLE
Milestone 6

### PR DESCRIPTION
**Pull request by Fueredoriku**
First milestone of our second year of development!
In this release we focused on onboarding of new members and some gameplay and design improvements.

New features!
- New map "Grand Canyon"
- Overhauled auction scene
- Improvements to "Crater Town" level design
- Minor bugfixes

### CODE REVIEW

**Review by toberge — APPROVED** [review]

**Comment by Fueredoriku** [comment]
Waiting for #250 before merge.

Branch has been built and tested.

**Pull request merged**
